src/nitypes/bintime/_timedelta.py

```diff
@@ -40,6 +40,13 @@
 _OtherDateTime: TypeAlias = Union[dt.datetime, ht.datetime]
 
 
+class WholeAndFractionalSeconds(NamedTuple):
```

Comment:
Move the WholeAndFractionalSeconds tuple to a separate file. In a file named _timedelta I would expect TimeDelta to be the first (and possibly only object). See [SampleIntervalMode](https://github.com/ni/nitypes-python/blob/e2679853b3fd40dee88fc91bd39a143dc62edbce/src/nitypes/waveform/_timing/_sample_interval/_mode.py#L6)for an example.